backend/api/Areas/Documents/DocumentController.cs

```diff
@@ -87,11 +87,38 @@ public IActionResult GetRelationshipDocuments(DocumentRelationType relationshipT
         }
 
         /// <summary>
-        /// Deletes the specific document relationship for the given type.
+        /// Uploads a document for the given relationship.
         /// </summary>
         /// <param name="type">Used to identify document type.</param>
         /// <param name="entityId">Used to identify document's parent entity.</param>
         /// <returns></returns>
+        [HttpPost("upload/{relationshipType}/{parentId:long}")]
+        [Produces("application/json")]
+        //[HasPermission(Permissions.PropertyAdd)]
```

Comment:
we really need to enforce permissions here - there has been a couple PRs but not actual permissions in place